lib/report.js

```diff
@@ -62,8 +62,20 @@ class Report {
       try {
         const path = resolve(this.resolve, v8ScriptCov.url)
         const script = v8toIstanbul(path, this.wrapperLength)
-        script.applyCoverage(v8ScriptCov.functions)
-        map.merge(script.toIstanbul())
+
+        // https://github.com/nodejs/node/blob/v12.0.0/lib/internal/modules/esm/create_dynamic_module.js#L12-L20
+        if (!(
+          v8ScriptCov.functions.length === 3 &&
```

Comment:
The V8 coverage of the CJS-ESM bridge looks like:

```
[
  {
    functionName: '',
    ranges: [ [Object] ],
    isBlockCoverage: true
  },
  {
    functionName: 'get',
    ranges: [ [Object] ],
    isBlockCoverage: false
  },
  {
    functionName: 'set',
    ranges: [ [Object] ],
    isBlockCoverage: true
  }
]
```

Reply:
this seems a little fragile ... also, if we need to use feature detection, let's pull the behavior into a new file, like `detect-cjs-esm-bridge.js` or something along those lines.

Reply:
Yes, it is fragile... but better than doing nothing.

> et's pull the behavior into a new file

Done. https://github.com/bcoe/c8/commit/f53068d64de3679637c13807a858b777c81cfba5